pontoon/base/models.py

```diff
@@ -67,6 +67,28 @@ def validate_cldr(value):
                 '%s must be a list of integers between 0 and 5' % value)
 
 
+def get_latest_activity(translations):
```

Comment:
The logic here is mostly fine, but I'm not a huge fan of functions living in `models.py` when there's more interesting places to hook this stuff in.

Because the only parameter is a translation queryset, I think making a [custom Translation queryset](https://docs.djangoproject.com/en/1.8/topics/db/managers/#calling-custom-queryset-methods-from-the-manager) is a good idea. Then we can take _any_ translation queryset and just call `translations.latest_activity()` and get back the object we want. Thoughts?